File: Array/1FindMin.cpp
#include <iostream>
#include <climits>
using namespace std;

int findMin(int array[])
{
    int min = INT_MIN;
    for (int i = 0; i < 5; i++)
    {
        if (array[min] < array[i])
        {
            min = array[i];
        }
    }
    return min;
}

int main()
{
    int arr[5] = {1, 9, 2, 4, 0};
    cout << findMin(arr) << endl;
}

File: Array/1FindMin.js
const arr = [9, 2, 1, 5, -1, 4, 0]
const findMin = (arr) => {
	var min = arr[0]
	for (let i = 0; i < arr.length; i++) {
		if (arr[i] < min) {
			min = arr[i]
		}
	}
	return min
}

console.log(findMin(arr))
